content/influxdb/clustered/install/deploy.md

```diff
@@ -9,6 +9,46 @@ menu:
 weight: 104
 ---
 
+Use Kubernetes and related tools to deploy your InfluxDB cluster.
+This guide provides instructions for deploying your InfluxDB cluster using the
+following tools:
+
+- **kubectl**: CLI for controlling the Kubernetes cluster manager
+- **kubit**: A Kubernetes controller that can render and apply jsonnet
+  templates based on the [kubecfg](https://github.com/kubecfg/kubecfg) jsonnet
+  tooling and framework
+
+InfluxDB Clustered uses an `AppInstance` Kubernetes custom resource (CR) to
+configure and deploy your InfluxDB Cluster.
+Installing a CR requires cluster-wide permissions and may cause `kubectl` to
+fail if you do not have those permissions in your cluster.
+
+`kubit` uses your local `kubectl` credentials to install CRs.
+This still requires you to have permissions to install cluster-wide resources
+(CRs), but doesn’t require `kubit` to be installed with those permissions
+in the cluster. 
+
+{{% note %}}
+**If you meet any of the following criteria, [install and use the `kubit` CLI](https://github.com/kubecfg/kubit#cli-tool)
+on your local machine to use it as an operator from your local terminal and
+deploy your cluster:**
+
+- You do not have permissions to install a CR.
+- You do not have permissions to install the operator in the `kubit` namespace.
+- You do not have permissions to create cluster-wide role-based access
+  control (RBAC).
+- You want to preview the generated YAML.
+- You do not want to run the operator in your InfluxDB cluster.
```

Comment:
```suggestion
- You do not want to run the operator in your Kubernetes cluster.
```

I believe this could be a typo, since it is a K8s cluster where the operator is installed rather than an InfluxDB cluster.

I might be missing some specific context though too :smile: Do we refer to a customer's K8s cluster as their "InfluxDB cluster"? Ignore me if so.